### PR TITLE
perf(open_graph): only require cheerio when content contains '<img'

### PR DESCRIPTION
**Pull request by SukkaW**
## What does it do?

only require cheerio when content contains '<img'

## How to test

```sh
git clone -b BRANCH https://github.com/USER/hexo.git
cd hexo
npm install
npm test
```

## Screenshots

https://github.com/hexojs/hexo/issues/3663#issuecomment-521629413

## Pull request tasks

- [ ] Add test cases for the changes.
- [x] Passed the CI test.

### CODE REVIEW

File: lib/plugins/helper/open_graph.js [hunk 1]
```diff
@@ -59,12 +58,16 @@ function openGraphHelper(options = {}) {
   if (!images.length && content) {
```
Comment (curbengh) [inline lib/plugins/helper/open_graph.js]: Is `if (!images.length && content && content.includes('<img'))` safe? I'm wondering if `content` can be null. Let me try with an empty file.

Comment (SukkaW) [reply lib/plugins/helper/open_graph.js]: Maybe some syntax error might cause content to be null, I'm not sure.

**Comment by curbengh** [comment]
My preference is on https://github.com/hexojs/hexo/pull/3680

**Comment by SukkaW** [comment]
Superseded by #3680

**Pull request closed without merging**